pytorch_lightning/profiler/pytorch.py

```diff
@@ -195,12 +195,13 @@ class PyTorchProfiler(BaseProfiler):
         "test_step",
         "predict_step",
     }
+    RECORD_FUNCTION_PREFIXE = "optimizer_step_and_closure_"
```

Comment:
typo in prefix here?

Reply:
Thanks! Fixed here: https://github.com/PyTorchLightning/pytorch-lightning/pull/8308

Reply:
thanks for following up on it so quickly! cheers